spark/v3.2/build.gradle

```diff
@@ -213,6 +213,7 @@ project(":iceberg-spark:iceberg-spark-runtime-${sparkMajorVersion}_${scalaVersio
       exclude group: 'commons-logging', module: 'commons-logging'
     }
     implementation project(':iceberg-hive-metastore')
+    implementation project(':iceberg-dell')
```

Comment:
Adding this module to the Spark runtime should be discussed and done in a separate PR, not in a documentation PR. Can you bring this up on the dev list and summarize what it pulls in and changes?

You'll also need to update the LICENSE and NOTICE files.